packages/ui/lib/components/TrackInfo/index.tsx

```diff
@@ -4,6 +4,15 @@ import { Icon } from 'semantic-ui-react';
 import Cover from '../Cover';
 import styles from './styles.scss';
 
+const CopyTextToClipboard = async () => {
+  const track_element = document.getElementById('track_name');
+  try {
+    await navigator.clipboard.writeText(track_element.innerHTML);
```

Comment:
This shouldn't be done this way in React. Instead, you can get the title from the redux store.